src/h5web/visualizations/heatmap/HeatmapToolbar.tsx

```diff
@@ -37,7 +37,7 @@ function HeatmapToolbar(): ReactElement {
         />
       )}
 
-      <Separator />
+      {dataDomain && <Separator />}
```

Comment:
> the responsive toolbar requires the toolbar controls to be direct children.

I think this ought to be put somewhere in comments. Otherwise, I might end up in trouble in a few weeks because I didn't remember this :sweat_smile:

Reply:
Yeah, TS won't let me forbid fragments... I'm going to throw somewhere in the code of the Toolbar, I think it's the only way to make this error discoverable.